### DISCUSSION TITLE
Restore suspended account?

### DISCUSSION THREAD

**Discussion by BurntNorton42** [mastodon/mastodon#20004] [Q&A]
Hi, I created a mastodon account with the intention of moving from twitter a while ago. But... I didn't put the effort in to keep on top of understanding how Mastodon worked and didn't access the forum for over 12 months. I understand in terms of the TOC that lead to my account being suspended. 

However... with the (almost certain) demise of Twitter, I want to seriously move over to Mastodon. 

Is there any way to get my account reinstated? If so, who should I get in touch with for this?

Thanks

**Comment by trwnh** [comment]
Contact the admin of the website you signed up on
